README.md

```diff
@@ -258,6 +259,10 @@ See also [Games Made With Piston](https://github.com/PistonDevelopers/piston/wik
 * [Veloren](https://gitlab.com/veloren/veloren) — An open world, open source multiplayer voxel RPG game currently in alpha development [![build badge](https://gitlab.com/veloren/veloren/badges/master/pipeline.svg)](https://gitlab.com/veloren/veloren/-/pipelines)
 * [Zone of Control](https://github.com/ozkriff/zoc) — A turn-based hexagonal strategy game [![build badge](https://api.travis-ci.org/ozkriff/zoc.svg?branch=master)](https://travis-ci.org/ozkriff/zoc)
 
+### Graph theory
```

Comment:
Not sure if this can be placed in Macro or somewhere else, but no issues anyway

Reply:
Mh, I don't think it fits in with the crates listed under `Macro`. We can rename the category to `Graph algorithms`. I didn't want to use just `Graphs` as it's often mistaken as graph diagrams.

Reply:
Maybe "algorithms" sounds better

Reply:
As in `Graph algorithms` ?